kernel/combinatorics/hdegree.cc

```diff
@@ -1030,6 +1030,10 @@ void scComputeHC(ideal S, ideal Q, int ak, poly &hEdge, ring tailRing)
     hComp(hexist, hNexist, k, hexist, &hNstc);
   else
     hNstc = hNexist;
+  if (hNexist==0)
```

Comment:
Why this? If hNexist would be 0, this routine must not be called anyway:
hEdge would be undefined!
Proposal: change to assume(hNexist>0);